images/linux/Ubuntu2004-README.md

```diff
@@ -170,7 +170,7 @@
 
 ### Databases
 - MongoDB 5.0.2 (apt source repository: https://repo.mongodb.org/apt/ubuntu)
-- Postgre SQL 13.4 (apt source repository: https://apt.postgresql.org/pub/repos/apt/)
+- PostgreSQL 13.4 (apt source repository: https://apt.postgresql.org/pub/repos/apt/)
```

Comment:
Please don't update the autogenerated documentation files. We should fix  - https://github.com/actions/virtual-environments/blob/7fe7393aa2e80159549986f9980329cc0a3651ef/images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1#L4